JDI.Light/JDI.Light/Elements/Composite/MultiDropdown.cs

```diff
@@ -0,0 +1,138 @@
+using System.Text;
+using System.Threading.Tasks;
+using JDI.Light.Attributes;
+using JDI.Light.Elements.Common;
+
+namespace JDI.Light.Elements.Composite
+{
+    public class MultiDropdown : UIElement
+    {
+        [FindBy(Tag = "button")]
+        private Button _header;
+        
+        public List<MultiDropdownElement> Options
+        {
+            get
+            {
+                var elems = FindElements(By.XPath(".//li")).ToList();
+                var labels = FindElements(By.XPath(".//li//label")).ToList();
+                var chBoxes = FindElements(By.XPath(".//li//input")).ToList();
+                List<MultiDropdownElement> elementList = new List<MultiDropdownElement>();
+                for (int i = 0; i < elems.Count; i++)
+                {
+                    elementList.Add(new MultiDropdownElement(By.XPath(".//li"),labels[i],chBoxes[i], elems[i]));
+                }                
+                return elementList;
+            }
+        }
+
+        public bool OptionIsEnabled(string name)
+        {            
+            return Options.Any(x => x.OptionIsEnabled && x.Text == name);
+        }
+
+        public void SelectOptionByname(string name)
+        {
+            Expand();
+            Options.FirstOrDefault(x => x.Text == name).Select();
+            Close();
+        }
+
+        public List<string> GetSelectedOptions()
+        {            
+            return Options.Where(x => x.Selected).Select(x=>x.Text).ToList();
+        }
+
+        public bool OptionsAreSelected(List<string> options)
+        {
+            return (Options.Where(x=>x.IsSelected).All(x => options.Contains(x.Text)) && options.All(x => Options.FirstOrDefault(y => y.Text == x).IsSelected));
+        }
+
+        public void SelectOptions(List<string> options)
+        {
+            Expand();
+            foreach(var option in options)
+            {
+                Options.FirstOrDefault(x => x.Text == option).Select();
+            }
+            JsExecutor.ExecuteScript("arguments[0].scrollIntoView();", WebElement);
+            Close();
+        }
+
+        public bool OptionExists(string option)
+        {
+            return Options.Any(x => x.Text == option);
+        }
+
+        public MultiDropdown(By locator) : base(locator)
+        {
+        }
+
+        public void Expand()
+        {
+            if (!GetAttribute("class").Contains("open"))
+            {
+                _header.Click();
+            }
+        }
+
+        public void Close()
+        {
+            Expand();
+        }
+
+        public class MultiDropdownElement : UIElement
+        {
+            [FindBy(Tag = "label")]
+            private IWebElement _label;
+
+            [FindBy(Tag = "input")]
+            private IWebElement _checkBox;
+
+            public bool IsSelected
+            {
+                get
+                {
+                    return (GetAttribute("class") == "active");
+                }
+            }
```

Comment:
get-only property can be made simplier:
![image](https://user-images.githubusercontent.com/6689884/54747401-da135100-4bdf-11e9-8324-220cab712f8c.png)